routing/router.go

```diff
@@ -23,6 +24,27 @@ func NewRouter(conf *config.Config) *mux.Router {
 			Handler(route.HandlerFunc)
 	}
 
+	router.PathPrefix("/jaeger/{rest:.*}").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can you please add some comments what this is doing and why, as it is so much different from the other routes?

Reply:
All requests from the frontend to the backend with prefix /jaeger. It's like a proxy @pilhuhn.
All request to /jaeger go to http://jaeger-query-istio-system.127.0.0.1.nip.io/api/

For example if someone query :
/jaeger/**services**  ==> http://jaeger-query-istio-system.127.0.0.1.nip.io/api/services

/jaeger/**traces?end=1521531900000000&limit=20&lookback=1h&maxDuration&minDuration&service=jaeger-query&start=1521500400000000**

this call to ==>/api/**traces?end=1521531900000000&limit=20&lookback=1h&maxDuration&minDuration&service=jaeger-query&start=1521500400000000**

Reply:
I did not meant "explain it to me", but rather "put a comment in the code" :-)

Reply:
Sorry @pilhuhn I was finish the PR in the UI aaht depends of this and I not read well your comment